spec/classes/collectd_init_spec.rb

```diff
@@ -9,20 +9,13 @@
 
       options = os_specific_options(facts)
       context 'with all defaults' do
-        it { is_expected.to contain_class('collectd') }
```

Comment:
why did you remove those two?

Reply:
When inside `init.pp` we know that a class called collectd will always be there. I just didn't see any value to the test.
Same with params.